extractor/src/main/java/org/schabi/newpipe/extractor/StreamingService.java

```diff
@@ -282,6 +290,27 @@ public CommentsExtractor getCommentsExtractor(final String url) throws Extractio
     // Utils
     //////////////////////////////////////////////////////////////////////////*/
 
+    public final String getIdByUrl(final String url) throws ParsingException{
```

Comment:
Instead of adding this function just to use it for search URLs in the client, I think a better option is what I suggested in the client review